nft-minter/src/nft_minting.rs

```diff
@@ -71,9 +71,16 @@ pub trait NftMintingModule:
             "May not mint after deadline"
         );
 
+        let caller = self.blockchain().get_caller();
+        if current_timestamp < brand_info.whitelist_expire_timestamp {
```

Comment:
So we set the whitelist expire timestamp to 0 in case we don't have a whitelist.

Reply:
Yep, or just the current timestamp, either work.